onlineconf.go

```diff
@@ -228,6 +236,72 @@ func (m *Module) GetBool(path string, d ...bool) bool {
 	}
 }
 
+// GetStruct reads a structured value of a named parameter from the module.
+// It stores this value in the value pointed by the value argument
```

Comment:
Looks confusing.

What do you think about this?
```go
// It caches unmarshalled object. Unmarshalled value must not be modified outside.
```

Reply:
Caching of modifiable objects is not safe. GetStruct should be used carefully.

90% of cases you don't need to cache arbitrary structs. But `map[string]string` or `map[string]int` could be useful more often. And safe interface for this cases could be suitable. What do you thing about [romaps](https://github.com/mrsndmn/onlineconf-go/blob/onlineconf-go-safe-clean-hist-maps/romap.go)?

Reply:
`GetStruct()` doesn't care about the type of a value it reads. The only requirement to this type is to be unmarshalable from JSON. It is the responsibility of a developer to use correct types, safe if they prefer. Of course, description of this method should be modified to mention what the value must conform to `json.Unmarshaler` and will be cached.